src/app/routes/api/v3/admin/forms/__tests__/admin-forms.preview.routes.spec.ts

```diff
@@ -413,236 +343,9 @@ describe('admin-form.preview.routes', () => {
-          }),
-        )
-        .query({ captchaResponse: 'null' })
-
-      expect(response.status).toBe(400)
-      expect(response.body.message).toEqual('Validation failed')
-    })
-
-    it('should return 400 when response has both answer and answerArray', async () => {
-      const { form } = await dbHandler.insertEmailForm({
-        formOptions: {
-          hasCaptcha: false,
-          status: FormStatus.Public,
-          admin: defaultUser._id,
-        },
-        // Avoid default mail domain so that user emails in the database don't conflict
-        mailDomain: 'test2.gov.sg',
-      })
-
-      const response = await request
-        .post(`${SUBMISSIONS_ENDPT_BASE}/${form._id}/preview/submissions/email`)
-        .field(
-          'body',
-          JSON.stringify({
-            responses: [{ ...MOCK_TEXTFIELD_RESPONSE, answerArray: [] }],
-          }),
-        )
-        .query({ captchaResponse: 'null' })
-
-      expect(response.status).toBe(400)
-      expect(response.body.message).toEqual('Validation failed')
-    })
-
-    it('should return 400 when attachment response has filename but not content', async () => {
-      const { form } = await dbHandler.insertEmailForm({
-        formOptions: {
-          hasCaptcha: false,
-          status: FormStatus.Public,
-          admin: defaultUser._id,
-        },
-        // Avoid default mail domain so that user emails in the database don't conflict
-        mailDomain: 'test2.gov.sg',
-      })
-
-      const response = await request
-        .post(`${SUBMISSIONS_ENDPT_BASE}/${form._id}/preview/submissions/email`)
-        .field(
-          'body',
-          JSON.stringify({
-            responses: [omit(MOCK_ATTACHMENT_RESPONSE), 'content'],
-          }),
-        )
-        .query({ captchaResponse: 'null' })
-
-      expect(response.status).toBe(400)
-      expect(response.body.message).toEqual('Validation failed')
-    })
-
-    it('should return 400 when attachment response has content but not filename', async () => {
-      const { form } = await dbHandler.insertEmailForm({
-        formOptions: {
-          hasCaptcha: false,
-          status: FormStatus.Public,
-          admin: defaultUser._id,
-        },
-        // Avoid default mail domain so that user emails in the database don't conflict
-        mailDomain: 'test2.gov.sg',
-      })
-
-      const response = await request
-        .post(`${SUBMISSIONS_ENDPT_BASE}/${form._id}/preview/submissions/email`)
-        .field(
-          'body',
-          JSON.stringify({
-            responses: [omit(MOCK_ATTACHMENT_RESPONSE), 'filename'],
-          }),
-        )
-        .query({ captchaResponse: 'null' })
-
-      expect(response.status).toBe(400)
-      expect(response.body.message).toEqual('Validation failed')
-    })
   })
 
-  describe('POST /admin/forms/:formId/preview/submissions/encrypt', () => {
+  describe('POST /admin/forms/:formId/preview/submissions/storage', () => {
```

Comment:
i pushed commit directly to update the route in the description e529790

Reply:
thanks! sorry I missed that :(